### ISSUE TITLE
fanva

### ISSUE THREAD

**Issue by lynn** [CvGC/dict#34]
The old definition, for reference:

> x<sub>1</sub> translates text/utterance x<sub>2</sub> to language x<sub>3</sub> from language x<sub>4</sub> with translation result x<sub>5</sub>.

There was some talk in #lojban recently about:
1. Getting rid of x<sub>3</sub> and x<sub>4</sub>, in favor of **bau**.
2. Extending x<sub>2</sub> and x<sub>5</sub> to include poems, stories, books, songs, videos, …

.i pei?

**Comment by este-rs** [comment]
I think the x3 and x4 can stay. And I've been using x2 and x5 for
non-utterances all along. So giving that meaning may make life easier...
―uak

**Comment by eaburns** [comment]
Would bau be the to-language or the from?

**Comment by solpahi** [comment]
It is no secret that I would be more than happy to drop fanva3 and fanva4, but I do not suggest using **bau** instead. As your question highlights, **bau**, like most BAI, is horribly ambiguous.

The reasoning behind dropping fanva3 and fanva4 is that those places are part of fanva2 and fanva5. Let **fa'anva** be _x1 translates x2 into x3_:

```
mi fa'anva zoi gy. What's your name? .gy «lu ma cmene do li'u»
I translate 'What's your name?' into 'ma cmene do'.
```

Do you need **bau** here? The text _'What's your name?'_ is English in and of itself. Therefore, if you translate it, then you translate from English.

When the listener doesn't know, you can add **noi .inglico** to indicate that this text is English.

However, I understand that some people would prefer to keep **fanva** as is for compatibility reasons. In a lot of cases, using **smudu'i** would be my recommendation anyway (then there is no question of who the fanva1 is, and the awkward place shuffling seen so often with **fanva** becomes unnecessary).

**Comment by Ntsekees** [comment]
fanva ≈ x1 kanji lo ka ce'u vlamei su'o da lo bausmu be x2 bei x4 bei x1 fo x3

I don't like the idea of removing fanva-x3 and fanva-x4 as I consider them essential to the concept of translation, and am not aware of any good way to express them otherwise.

I don't find using {noi} any satisfying at all for specifying the languages, as a single letter or sound string might mean various things in various languages, and {noi} has no restrictive power.

{mi fanva zo mi lo jbobau lo turkybau zo xu}
{mi fa'anva fi zo xu fe zo mi noi valsi lo si'o mibypre fa'u ke nanba nenri marji ke'e fa'u se mipri mu'a kei lo jbobau fa'u lo fasybau fa'u lo jugbau}

**Comment by Ntsekees** [comment]
Having the source language in the argument structure might however be less important, as the translator will usually begin by identifying the language of the source text, and if they fail to do so they're unlikely to make any translation at all.

**Comment by eaburns** [comment]
But when translating, presumably you are translating to only one destination language, not various.

What is unclear about:
{mi fa'anva zo mi ne lo lojbo zo xu ne lo turkybau}

**Comment by Ntsekees** [comment]
> But when translating, presumably you are translating to only one destination language, not various.

Indeed, but the question is how to express what is the target language when that's desired (especially in the case where the translation result may correspond to more than one languages and you want to make clear which one it is). Another example is "How would you translate ‹hello› in Indonesian?", would you really say {da'i do fa'anva zoi gy. xelo .gy ma noi baxso}? I guess you'd rather have to go with a paraphrasis such as {da'i do fa'anva zoi gy. xelo .gy mo'oi vlamei be fi lo smuni be ri ku bei lo baxso}.

**Comment by eaburns** [comment]
I would've suggested {da'i do fa'anva zoi gy. xelo .gy ma noi baxso}, yes. I don't understand what's wrong with it.

**Comment by Ntsekees** [comment]
I'm not sure whether {ma noi} is meaningful, but if it is, I'd expect {sipna fa ma noi mlatu} to mean something like {ma sipna .i lo sipna cu mlatu}. (Please tell me if you have another interpretation.)
If so, then {do fa'anva zo mi su'o da na .a ma noi baxso} would mean {do fa'anva zo mi su'o da na .a ma .i lo poi'i do fa'anva zo mi su'o da na .a ke'a cu baxso}, "If you translate the word ‹mi›, you'd translate it to what? The thing X such that if you translate the word ‹mi›, you'd translate it to X, is such that X is Malay-Indonesian.", which doesn't seem to suit the intended meaning. That second sentence is a separate assertion that means "if you translate ‹mi›, you'll translate it to something Malay-Indonesian." (It's not even a command, but a claim.)

**Comment by Ntsekees** [comment]
Maybe {da'i do fa'anva zo mi ma noi ko'oi baxso} would work in practice, I'm not sure. But even so I don't feel it ensures that the requester does literally request the translation result to mean "mi" in baxso language, but merely that the translation result is incidentally baxso in some regard even if it the translation target language wasn't baxso. But I'm probably nitpiking there. :-)

**Comment by Ntsekees** [comment]
Let's imagine the following situation. Somebody asks {mi fa'anva zo do zo du .i ge zo do lojbo gi zo du dotco .i xu lo nu go'e cu drani}. If you both know German and Lojban, in context you can expect that the source language was Lojban and the target language was German, but yet it's not explicitly stated, what is said is just that "do" is incidentally Lojbanic and "du" is incidentally Germanic, which is true, but those two statements aren't explicitly related to the initial claim {mi fa'anva zo do zo du}, so you can't be sure. "do" could actually be from English and mean "female deer" (spelled "doe"), or be from Catalan and mean "talent", or be from Esperanto and mean "therefore". As for "du", it could be the Esperanto for "two", or the French for "soft" (if it's uttered phonetically instead of parallelling the way it's written in the source language).

The most careful answer might then be {lo smuni be zo do bei lo lojbo ja'a smuni zo du lo dotco} (without asserting the original *translation* was correct as you're not fully certain about the source and destination language), so the original requester will be able to judge for themself whether their translation was correct in the light of your reply.

mi'e nitpiki mu'o

**Comment by solpahi** [comment]
You don't have to use **ma noi**. You can use **mo'oi** (e.g. **fa'anva zo mi mo'oi dotco**). (I agree that **ma noi** is usually not _smudra_ in these contexts)

Anyway, to name a few more reasons in favor of a trimmed **fanva**, I generally find 5-place predicates rather excessive. My preference would be to only go beyond 3 when it is logically necessary (predicates that seem hard to reduce are things like **venfu**, **cnemu**, **sfasa** (where a reduction would be possible but would not obviously bring any improvements)). Another point that I find important is trying to keep the number of frames as low as possible. FANVA is currently a singleton frame, meaning there is no other gismu that parallels it. By trimming it to 3, the number of frames would be reduced by 1, which I would see as a good thing.
